src/components/03-components/02-alerts/alerts--block.njk

```diff
@@ -1,7 +1,7 @@
 <h3>SDS Block Alert</h3>
 <div class="sds-alert--block">
   <div class="sds-alert--block-icon grid-col-auto">
-    <i class="sds fa-exclamation fa-2x"></i>
```

Comment:
I noticed fa-2x missing throughout all instances including other files, not sure if this is by design

Reply:
This is in alerts.config